### ISSUE TITLE
Stuck on "start training"

### ISSUE THREAD

**Issue by Ninoharaa** [DeepLabCut/DeepLabCut#2660]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I tried many things : reduce frames, iterations, crop the video, max input size at 3000, global scale at 0.5... But now it just doesn't launch (on my iMac M3 chip) or on google colab T4 GPU). I managed to run a training though on a different local computer with a GPU and under the 2.3.6 version. I tried then to launch on Gogle Col with the 2.3.6 version but nothing is happening...

### Operating System

iOS 14.4.1 (23E224)

### DeepLabCut version

tried many : 2.3.10, 2.3.8, 2.3.6

### DeepLabCut mode

single animal

### Device type

GPU

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @Ninoharaa, I need more information to help you.

How did you install DeepLabCut? And how are you trying to run it? Are any error messages printed?

**Comment by Ninoharaa** [comment]
I followed the conda activation process on the DLC Github for the computer and for my MacBook with M3 chip I used the "DEEPLABCUT_M1.yaml". The only message I get is during the starting training phase : something closed to "You should try to reduce data size by modifying max input size and global scale". I tried to put global scale to 0.5, max input size to 1200 I think and cropped my frames (60 in total max). Then I ran on 2.3.8 / 9 and 6 and none of them worked.

**Comment by MMathisLab** [comment]
Hi @Ninoharaa welcome - how large are your frames, ie in pixels, 1080 x 480, etc?

**Comment by Ninoharaa** [comment]
Thank you for answering, they're 1912x418 (final size cropped)

**Comment by MMathisLab** [comment]
Okay great, then you need to set the limit above 1912, something like 2000 vs "max input size to 1200" ... the max input size is the max size an image that can be included in training.

But, that technically aside, you need a large gpu for such large images, something with 16GB or more of memory,

**Comment by Ninoharaa** [comment]
Noted, I will try with these parameters and probably pre-process my images, with maybe binning. Otherwise, I just noticed your update with possibility to use PyTorch, is there any difference in terms of efficiency for the training ? (I'm trying with another set of data with a powerful GPU and with 200 epochs).

**Comment by Ninoharaa** [comment]
Hello, I struggle now with another step : I tried to run a training with PyTorch. It reaches the 200 epochs but then when i want to analyze videos it says it only goes until 14th frame (on more than 1000). I assessed video corruption and tried to re encode but videos seem to not be corrupted. I runned DLC 3.0.0rc4 and 3.0.0rc5.

<img width="1099" alt="Screenshot-bug-analyze-videos2" src="https://github.com/user-attachments/assets/9531f985-6d2a-42aa-8524-6bdeb40cb94f">

Thanks for helping !
